docs/man/nng_http_req_get_uri.3http.adoc

```diff
@@ -20,7 +20,7 @@ nng_http_req_get_uri - return HTTP request URI
 #include <nng/nng.h>
 #include <nng/supplemental/http/http.h>
 
-const char *nng_http_req_get_method(nng_http_req *req);
+const char *nng_http_req_get_uri(nng_http_req *);
```

Comment:
You removed the name of the req parameter.  Can you please put this back, as the variable is referenced in the following paragraph.  Thanks.